ansibledoctor/templates/readme/_vars.j2

```diff
@@ -1,5 +1,12 @@
 {% set var = role.var | default({}) %}
 {% if var %}
+## Variables
```

Comment:
While I understand the change to add a requirements section, can you explain why you added this table here?

Reply:
My role contains hundreds of variables. Browser them all is a very painful toll.

A sum up variables array add some readability in roles' variables and internal links provided href to the variables section page.